packages/react/src/Dialog/Dialog.tsx

```diff
@@ -40,4 +38,9 @@ export const Dialog = forwardRef(
   ),
 )
 
-Dialog.displayName = 'Dialog'
+DialogRoot.displayName = 'Dialog'
+
+export const Dialog = Object.assign(DialogRoot, {
+  close: closeDialog,
+  open: (id: string) => (document.querySelector(id) as HTMLDialogElement)?.showModal(),
```

Comment:
Why are the functions defined in two different places?

Reply:
Yes, even if we don’t use them in the same way (yet), defining and exporting them in the same places is less confusing.